eyed/theme.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Eyed",
+    "version": "5.1",
+    "description": "Eyed theme for phpMyAdmin",
+    "author": "Febriansyah Riki Setiadi",
+    "url": "https://github.com/febryars33/eyed-phpmyadmin-theme/",
+    "supports": [
+        "5.1",
```

Comment:
5.1 works fine with the same css as 5.2
I have some doubts 🤔 ?

Reply:
that is true haha 😄
change to 5.1 ?

Reply:
It's a 5.1 only theme ?
On what version did you test it and develop it ?

Reply:
I use phpmyadmin 5.2

Reply:
Then it's best to support 5.2
I doubt your CSS can support both versions at a time given the differences between the versions :)

Reply:
I'll develop 5.2, that same with my phpmyadmin version. and i'll change support to 5.2
If css bugs found, open the issues on my repo
https://github.com/febryars33/eyed-phpmyadmin-theme/issues or this repo :D

what do you think?

Reply:
and i'll remove 5.1 from theme.json